curp/tests/common/curp_group.rs

```diff
@@ -76,7 +76,7 @@ impl CurpGroup {
             })
             .collect();
 
-        tokio::time::sleep(Duration::from_secs(2)).await;
+        tokio::time::sleep(Duration::from_secs(3)).await;
```

Comment:
How do we determine the waiting time? 3 seconds are really enough?

Reply:
Because the time a follower needs to wait before the election starts is 1~2 second. 3 seconds are almost 100% enough because vote split rarely fails and 3 seconds is enough for nearly 2 rounds of elections.